python/publish_unit_test_results.py

```diff
@@ -164,6 +164,21 @@ def get_var(name: str, options: dict) -> Optional[str]:
     """
     return options.get(f'INPUT_{name}') or options.get(name)
 
+def get_bool(name: str, options: dict, default: bool = False) -> Optional[str]:
+    """
+    Same as get_var(), but checks if the value is a valid boolean.
+    If the value is unset, defaults to false.
+    """
+    val = get_var(name, options).lower()
```

Comment:
`get_var` can return `None` if `name` does not exist in `options`, then calling `lower` fails

Reply:
https://github.com/EnricoMi/publish-unit-test-result-action/runs/4447581827?check_suite_focus=true#step:6:612